src/Http/Controllers/CP/Collections/CollectionsController.php

```diff
@@ -288,7 +288,8 @@ protected function createLinkBlueprint($namespace)
                 'title' => __('Link'),
                 'fields' => [
                     ['handle' => 'title', 'field' => ['type' => 'text']],
-                    ['handle' => 'redirect', 'field' => ['type' => 'link', 'required' => true]],
+                    ['handle' => 'redirect', 'field' => ['type' => 'link', 'required' => true, 'width' => '75']],
+                    ['handle' => 'redirect_http_status', 'field' => ['type' => 'select', 'required' => true, 'options' => [301, 302], 'width' => '25']],
```

Comment:
I wonder if it might be worth the options having labels like `301 (Permanent)` & `302 (Temporary)`? I don't imagine a lot of end-users will know the difference between the two.

Reply:
Can we make HTTP uppercase in the display label?

```suggestion
                    ['handle' => 'redirect_http_status', 'field' => ['type' => 'select', 'required' => true, 'options' => [301, 302], 'width' => '25', 'display' => __('Redirect HTTP Status')]],
```

Reply:
Yep - both done